criu/crtools.c

```diff
@@ -254,8 +255,6 @@ int main(int argc, char *argv[], char *envp[])
 		return 1;
 	}
 
-	util_init();
```

Comment:
I think we need to call it separately from cr_service_work. Each new service work should have an uniq criu_run_id and compel_run_id, because several service works can run concurrently.

Reply:
you are talking about this right? https://github.com/checkpoint-restore/criu/blob/6cfe7aa1142093f93a38a127a8a063e1470bca65/criu/cr-service.c#L1531

I've been digging a bit as well, looks like having it at the begginging of `cr_service_work` would be enough in all cases.

Reply:
After testing a little bit more I realized that we need to init in multiple places, once when  `cr_service_work` is forked and in the location I was already proposing in this patch.